disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/storage/files/utils/Constants.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.internal.storage.files.utils;
+
+import java.nio.charset.StandardCharsets;
+
+public final class Constants {
+
+  public static final byte[] NEW_LINE_BYTES =
```

Comment:
This is only used in a test. Is this a remnant from the json based serialzier?

Reply:
Nice catch, it is from when we were using the custom json serialization.